user_manual/groupware/calendar.rst

```diff
@@ -265,6 +265,13 @@ The ``Empty trash bin`` buttons will wipe all trash bin contents in one step.
 
 .. _calendar-working-hours:
 
+Automated User Status
+~~~~~~~~~~~~~~~~~~~~~
+
+When you have a calendar event scheduled that has a "BUSY" status, your user status will be automatically set to "In a meeting" unless you have set yourself to "Do Not Disturb" or "Invisible".
+You can overwrite the status with a custom message any time, or set your calendar events to "FREE".
+Calendars that are TRANSPARENT will be ignored.
```

Comment:
Don't you mean events?

Reply:
Well no there's apparently the option to set a calendar to TRANSPARENT as well

Reply:
`schedule-calendar-transp`

Reply:
I'd drop the all-caps to distinguish between icalendar properties and caldav ones.